R/mod-view-all-section.R

```diff
@@ -71,13 +71,29 @@ mod_view_all_section_server <- function(input, output, session, synapse, syn,
             levels = reorder_steps(unique(.data$step)),
             ordered = TRUE
           )
-        ) %>%
-        ## Arrange by submission and step
-        dplyr::arrange(.data$submission, .data$step)
+        )
 
       if (is.null(submissions)) {
         stop("No submissions found with requested status(es)")
       }
+      
+      ## Get data
```

Comment:
Why do we have to get this data a second time here? Can't we use the data we pulled back in mod-main, like we did for mod-review-section?

Reply:
Great point. I just updated it to pass in the pre-fetched metadata.